apps/web/src/app/pages/trade/bridge/_hooks/useWallets.tsx

```diff
@@ -28,24 +30,55 @@ const useWallets = (): {
         xChainId: evm.xChainId,
         disconnect: evm.disconnect,
       },
+      [XWalletType.EVM_ARBITRUM]: {
```

Comment:
make a different hook for your purpose and use it. because the hook is used in other parts. I'm not sure how the changed hook affect other parts.

Reply:
this is actually your change from here https://github.com/balancednetwork/balanced-network-interface/pull/1357/files#diff-f256bd85f7827a38d24b6001cc8517fdb676e0b7c6867036c60abe99b4a824e4

I think we are fine to keep this as is